docs/linuxoci.md

```diff
@@ -35,7 +35,7 @@ grafana-cli plugins install oci-datasource
 The plugin will be installed into your Grafana plugins directory, which by default is located at /var/lib/grafana/plugins. [Here is more information on the CLI tool](http://docs.grafana.org/plugins/installation/).
 
 ### Manually installation 
```

Comment:
This should be "Manual installation"

Reply:
Thanks for bringing this up. Just noticed